components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -341,6 +351,48 @@ To select the environment:
 
 {{< figure src="/images/automate/update-node-environment.png" alt="Update Node Environment">}}
 
+##### Run List
+
+You can edit, expand or collapse the run list of a node.
+
+{{< figure src="/images/automate/run-list-of-a-node.png" alt="Run List of a Node">}}
+
+Chef Infra allows you to edit the run list of a node by following the steps below:
+
+1. Select **edit** option. A popup window opens.
+
+2. The left section the window contains the list of environment of the nodes. Select any one environment from the list.
+
+3. The right section of the window will show the current run list of the selected environment.
+
+4. Select the run list you want for the specific environment and select **Create**.
+
+The popup window looks like as shown below:
+
+{{< figure src="/images/automate/edit-run-list.png" alt="Edit Run List">}}
+
+Chef Infra allows you to edit the run list of a node using a shortcut. Select the option **Edit Run List** from the ellipses icon of the specific node in the node list. Selecting the option will open the same popup window which can be used to edit the run list of a node.
+
+The ellipses icon list looks like:
+
+{{< figure src="/images/automate/edit-run-list-ellipses-icon.png" alt="Edit Run List Ellipses Icon">}}
```

Comment:
This image isn't very helpful in understanding what the ellipses is. I think it is better to remove it.